src/Employer/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -34,25 +27,7 @@ public async Task<IActionResult> Dashboard([FromRoute]string employerAccountId,
             if (TempData.ContainsKey(TempDataKeys.DashboardInfoMessage))
                 vm.InfoMessage = TempData[TempDataKeys.DashboardInfoMessage].ToString();
 
-            if (string.IsNullOrEmpty(statusFilter))
-                statusFilter = Request.Cookies[CookieNames.VacancyStatusFilter];
-
-            ManageDashboardFilter(statusFilter, vm);
-
             return View(vm);
         }
-
-        private void ManageDashboardFilter(string statusFilter, DashboardViewModel vm)
-        {
-            if (!string.IsNullOrEmpty(statusFilter) && Enum.TryParse<VacancyStatus>(statusFilter, out var filter))
-            {
-                vm.Filter = filter;
```

Comment:
If you are removing this cookie could you please speak with Faizal/Raina/MA about having the cookie removed from their cookies and policies page please?

Reply:
Faizal on the case